set.mm

```diff
@@ -708474,6 +708599,38 @@ but which is a consequence of that a contradiction implies anything (see
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+    Integer powers - extension
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $( If an integer exponentiation with a positive mantissa is greater than 1,
```

Comment:
An integer power with...
(same below)

To me, these two are worth going to Main.

Reply:
OK, according to Wikipedia "Exponentiation" or Bronshtein "Handbook of Mathematics": "Exponentiation" is the operation of raising a "base" x to the power of the "exponent" y, resulting in the "power" x^y, also called "x to the power of y". In German: Das "Potenzieren" ist die Operation, eine "Basis" x mit einem "Exponenten" (oder auch einer "Hochzahl") y zu potenzieren. Das Ergebnis ist die "Potenz" (oder auch der "Potenzwert") x^y, auch mit "x hoch y" bezeichnet. 

Unfortunately, the "exponent" is sometimes also called "power" (see also Bronshtein "Handbook of Mathematics"!).

Therefore, "integer exponentiation" is the operation of raising a "base" x to the power of an integer y, resulting in the "integer power" x^y.

By the way, the term "mantissa" is used in the context of logarithms, isn't it? In the context of exponentiations, "base" should be used, shoudn't it?

As a consequence, "integer exponentiation" must be replaced by "integer power" in many places in set.mm, e.g., ~expeq0 ("Positive integer exponentiation is 0 iff its mantissa is 0" => "A positive integer power is 0 iff its base is 0."). Furthermore, statements like "raised to a negative integer power" are also not precise according this terminology (it shoud read "raised to the power of a negative integer").

If this is the agreed terminology, it should be explained at the beginning of section" Integer powers".

Reply:
Yes, these are the meanings of base / exponent / power / exponentiation I learned in middle school (in French) and they are pretty standard. I didn't know the term "mantissa".

I think there is no urging need to make global changes, though. The terms power/exponent are sometimes confused with each other, but context allows the reader to know which is meant.

You may move your two results to Main in my opinion.

Reply:
One quirk which may affect the terminology is that we have three notations https://us.metamath.org/mpeuni/df-exp.html and https://us.metamath.org/mpeuni/df-ef.html and https://us.metamath.org/mpeuni/df-cxp.html which sometimes give the same values ( https://us.metamath.org/mpeuni/efzval.html and https://us.metamath.org/mpeuni/ecxp.html ) but in general are basically different things. The need to make this distinction is mostly metamath specific, although other texts may have something similar until they sweep it all under the rug with "since blah blah blah we can identify (notation one) with (notation two)").

As for mantissa, the usages in set.mm which I noticed seemed to be misusing the term, at least in the sense that https://en.wikipedia.org/wiki/Common_logarithm#Mantissa_and_characteristic and https://mathworld.wolfram.com/Mantissa.html and other web pages I found seemed to agree on what it is, and it doesn't seem to be the base.

Having said all that, I agree there isn't an urgent need to make any global changes.

Reply:
Terminology adjusted (and documented).
2 theorems moved to main.